src/components/widgets/TreeView/TreeViewLeaf.js

```diff
@@ -43,6 +44,24 @@ const TreeViewLeaf = ({
           </em>
         </small>)
       </span>}
+    {organizational &&
+      <OverlayTrigger
+        placement="bottom"
```

Comment:
Do we have any other `placement="bottom"`? I'd prefer `"top"` if it will look decent.